code/datums/gamemode/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -14,7 +14,7 @@
 	required_candidates = 1
 	weight = 7
 	cost = 10
-	requirements = list(40,30,20,10,10,10,10,10,10,10)
+	requirements = list(10,20,10,10,10,10,10,10,10,10)
```

Comment:
Why not just make it a flat (10, ...) or (20, 20, 10...)?

Why does population level 2 have a bump?